package.json

```diff
@@ -167,6 +176,7 @@
     "rollup-plugin-node-resolve": "5.2.0",
     "rollup-plugin-sass-variables": "0.1.1",
     "rollup-plugin-vue": "5.0.1",
+    "serve": "11.1.0",
```

Comment:
NOTE: Serve is a dependency we need but it wasn't explicit at devDep. Added.

Reply:
I'm taking it easy today so I haven't had time to play with this locally to see if `serve` or `start-server-and-test` are truly needed, but in this PR as you have it `serve` isn't being used at all, so it shouldn't be added back as a dependency. Again: we must remove as many dependencies as possible, and only add dependencies if it seems like the costs of *not doing so* significantly outweigh the costs of doing so (as may be the case with cypress).

We currently have ~1000 dependencies:

```
-> % ls node_modules | wc -l
     943
```

Each time a new dependency is added, or an old dependency is updated, is an opportunity for everything to go disastrously wrong. Until we switch away from NPM and Node.js to something sane, we have to be extremely careful here.

Reply:
`serve` dep removed then.